.prow.yaml

```diff
@@ -23,7 +23,7 @@ presubmits:
     cluster: default
     spec:
       containers:
-        - image: "nephio/linter:1"
+        - image: "nephio/linter:latest"
```

Comment:
it might be good idea to use a fixed version instead of the :latest tag, what do you think?

Reply:
You're absolutely right, in this case we don't really anticipate this image to change very often, so for simplicity 'latest' will do. This is also 'alias' to numbered image so in any case it's easy to backtrace image version based on date of job execution.

Reply:
@vjayaramrh you have a valid point, but I think this Docker image has only context on the test-infra CI, so I don't expect any external project consuming it. Therefore, it's not necessary to keep a history of the previous versions.